firebase.json

```diff
@@ -192,6 +192,7 @@
       { "source": "/go/text-field-characters", "destination": "https://docs.google.com/document/d/1OOFW0PEZf0orBl445YXJ3nfjyKOqZdmeQ6KU3glUh7s/edit?usp=sharing", "type": 301 },
       { "source": "/go/system-mouse-cursor", "destination": "https://docs.google.com/document/d/1bJLRy6flZ0wDCbpl2QA8SURUWXIvRJKMRRemxlOo1cA/edit?usp=sharing", "type": 301 },
       { "source": "/go/unobstructed-platform-views", "destination": "https://docs.google.com/document/d/1PCTnBoNYDTQgYOzoZw43Iu5zG5q7j6cliCDT3mhhqZw/edit?usp=sharing", "type": 301 }
+      { "source": "/go/flutter-platform-idling", "destination": "https://docs.google.com/document/d/1RhSluOcHED6wEckY8CGK1mowRcWGAEGWraN6REPaORs/edit?usp=sharing", "type": 301 }
```

Comment:
@chinmaygarde, the build will fail unless you add a comma to the second-to-last line. All lines must end in comma EXCEPT for the last line.